README.md

```diff
@@ -1,39 +1,31 @@
 # react-native-auth0
 
-React Native toolkit for Auth0 API, compliant with [RFC 8252](https://tools.ietf.org/html/rfc8252)
-
 [![Build Status][circleci-image]][circleci-url]
 [![NPM version][npm-image]][npm-url]
 [![Coverage][codecov-image]][codecov-url]
 [![License][license-image]][license-url]
 [![Downloads][downloads-image]][downloads-url]
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Freact-native-auth0.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Freact-native-auth0?ref=badge_shield)
 
-## Important Notices
+📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • ⏭️ [Next Steps](#next-steps) • 💬 [Feedback](#feedback)
```

Comment:
Do these emoji's work when published to npm or are should we use the below suggestion instead?

```suggestion
:books: [Documentation](#documentation) • :rocket: [Getting started](#getting-started) • :next_track_button: [Next steps](#next-steps) • :speech_balloon: [Feedback](#feedback)
```

Reply:
The emoji are Unicode encoded so it should be the only format that will work in the most cases. For example, using the shortforms doesn't work in VS Code

<img width="845" alt="Screenshot 2022-10-20 at 12 21 12" src="https://user-images.githubusercontent.com/15910425/196923480-d8edfb91-954e-4577-aee9-d25e9a8eb5d0.png">

I am checking whether NPM supports this format.